### DISCUSSION TITLE
Where do I document changes to my fork

### DISCUSSION THREAD

**Discussion by JavaJoeS** [eclipse-platform/eclipse.platform#1205] [Q&A] [answered]
I can create a wiki from my fork, not sure where it goes or how to get to it..  WHERE is the best place to document what I am doing?
TIA

**Comment by mickaelistria** [comment, marked as answer]
If the documentation is for you, then the best is to create it in your own fork of the repo.
If the documentation is for reviewers, then it's to be part of the code or of the pull request comments.
If it's for end-users, it must be added to the official project documentation: https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/tree/master/eclipse.platform.common/bundles/org.eclipse.platform.doc.user
